src/applications/disability-benefits/2346/components/Accessories.jsx

```diff
@@ -151,7 +151,14 @@ class Accessories extends Component {
           accessorySupplies.map(accessorySupply => (
             <div
               key={accessorySupply.productId}
-              className="vads-u-background-color--gray-lightest vads-u-padding--3 vads-u-margin-y--3 accessory-page"
+              className={`vads-u-background-color--gray-lightest vads-u-padding--3 vads-u-margin-y--3 accessory-page ${
```

Comment:
We should extract this to a variable for clarity - having this function on the left side of the ternary is harder to read

Reply:
Do you mean put the shared classes in a variable but keep the ternary for the specific cases?

Reply:
do something like this
```
const hasSelectedProducts = selectedProducts.find(
                  selectedProduct =>
                    selectedProduct.productId === accessorySupply.productId
                )
```
then in your ternary 
```
{ hasSelectedProducts ? 'cssClass' : '' } 
```

Reply:
Okay I updated it to have the function be in a variable which returns a boolean and then placed that variable in my ternary.

Reply:
Oooh I just saw that we use classnames in this project - optional refactor would be to use the classnames helper - it would be quite a bit cleaner

Reply:
Just refactored to use classnames :slightly_smiling_face: